contracts/adapters/AaveV3LeverageStrategyExtension.sol

```diff
@@ -19,9 +19,15 @@
 pragma solidity 0.6.10;
 pragma experimental ABIEncoderV2;
 
-import {AaveLeverageStrategyExtension} from "./AaveLeverageStrategyExtension.sol";
+import { Math } from "@openzeppelin/contracts/math/Math.sol";
 
+import {AaveLeverageStrategyExtension} from "./AaveLeverageStrategyExtension.sol";
 import {IBaseManager} from "../interfaces/IBaseManager.sol";
+import { IChainlinkAggregatorV3 } from "../interfaces/IChainlinkAggregatorV3.sol";
+import {IPool} from "../interfaces/IPool.sol";
+import {IPoolAddressesProvider} from "../interfaces/IPoolAddressesProvider.sol";
+import { DataTypes } from "../interfaces/Datatypes.sol";
+import { PreciseUnitMath } from "../lib/PreciseUnitMath.sol";
```

Comment:
Formatting nits

Reply:
Fixed with a bunch of other formatting / documentation related changes in this Commit:
https://github.com/IndexCoop/index-coop-smart-contracts/pull/142/commits/3b5c204925aca0c0c31ee847d2788372b404ab7a